docs/docs-beta/docs/guides/etl/transform-dbt.md

```diff
@@ -1,4 +1,120 @@
 ---
 title: Transforming data with dbt
 sidebar_position: 20
----
\ No newline at end of file
+last_update:
+  date: 2024-08-26
+  author: Nick Roach
+---
+
+Dagster orchestrates dbt alongside other technologies, so you can schedule dbt with Spark, Python, etc. in a single data pipeline. Dagster's asset-oriented approach allows Dagster to understand dbt at the level of individual dbt models.
+
+## What you'll learn
+
+- How to import a basic dbt project into Dagster
+- How to set upstream and downstream dependencies on non-dbt assets
+
+---
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- a basic understanding of dbt, DuckDB, and Dagster concepts such as assets and resources
+- the dbt and DuckDB CLIs installed
+- Dagster, DuckDB, `plotly`, `dagster-dbt`, and `dbt-duckdb` Python packages installed
+</details>
+
+---
+
+## Setting up a basic dbt project
+
+Start by scaffolding a basic dbt project:
+
+```bash
+dbt init jaffle_shop
+```
+
+Make sure to select **DuckDB** as your database when prompted. This will create a dbt project with the following structure:
+
+```
+├── README.md
+├── analyses
+├── dbt_project.yml
+├── logs
+├── macros
+├── models
+│   └── example
+│       ├── my_first_dbt_model.sql
+│       ├── my_second_dbt_model.sql
+│       └── schema.yml
+├── seeds
+├── snapshots
+└── tests
+```
+
+If you try to build this example as is, it would immediately fail due to a non-null test not passing in `my_first_dbt_model.sql`. To fix this, you'll uncomment the last line in the model to filter out the null record:
```

Comment:
### Note to other reviewers

Nick and I chatted about this. It turns out the default dbt project here comes with a failing test :| He's going to throw together a mini dbt project with our models that users can scaffold, and then he'll update the instructions. 

TL:DR; This is weird/annoying and we're aware of + working on it